examples/usecases/ranking_with_multitask_learning.ipynb

```diff
@@ -0,0 +1,1663 @@
+{
```

Comment:
<p>Is the benefit to require less computational resources or are there additional benefits like higher accuracy in some targets?</p><p> </p><p>Can we add the real word (+reference) for MMoE, CGC and PLE? I do not know what the abbreviations are :)</p><p> </p><p>E.g. [Multi-gate Mixture-of-experts (MMOE)](link), etc.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/948/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Sure. I have clarified those points now</p>